From f47e513dcd8557b84463379d52d7156adef121a9 Mon Sep 17 00:00:00 2001
From: "Xiang, Haihao" <haihao.xiang@intel.com>
Date: Fri, 24 Jun 2016 10:04:28 +0800
Subject: Update PCI IDs for Kabylake

Remove unused PCI IDs and add new PCI IDs for KBL, the IDs are taken
directly from intel-gfx patches, which are under review:

https://lists.freedesktop.org/archives/intel-gfx/2016-June/099263.html
https://lists.freedesktop.org/archives/intel-gfx/2016-June/099264.html

Signed-off-by: Xiang, Haihao <haihao.xiang@intel.com>

diff --git a/src/i965_pciids.h b/src/i965_pciids.h
index fe3f4d0..1ea3c98 100644
--- a/src/i965_pciids.h
+++ b/src/i965_pciids.h
@@ -174,14 +174,12 @@ CHIPSET(0x591E, kbl, kbl,       "Intel(R) Kabylake")
 CHIPSET(0x5912, kbl, kbl,       "Intel(R) Kabylake")
 CHIPSET(0x5917, kbl, kbl,       "Intel(R) Kabylake")
 CHIPSET(0x5902, kbl, kbl,       "Intel(R) Kabylake")
-CHIPSET(0x5932, kbl, kbl,       "Intel(R) Kabylake")
 CHIPSET(0x591B, kbl, kbl,       "Intel(R) Kabylake")
-CHIPSET(0x592B, kbl, kbl,       "Intel(R) Kabylake")
 CHIPSET(0x593B, kbl, kbl,       "Intel(R) Kabylake")
 CHIPSET(0x590B, kbl, kbl,       "Intel(R) Kabylake")
 CHIPSET(0x591A, kbl, kbl,       "Intel(R) Kabylake")
-CHIPSET(0x592A, kbl, kbl,       "Intel(R) Kabylake")
-CHIPSET(0x593A, kbl, kbl,       "Intel(R) Kabylake")
 CHIPSET(0x590A, kbl, kbl,       "Intel(R) Kabylake")
 CHIPSET(0x591D, kbl, kbl,       "Intel(R) Kabylake")
-CHIPSET(0x593D, kbl, kbl,       "Intel(R) Kabylake")
\ No newline at end of file
+CHIPSET(0x5908, kbl, kbl,       "Intel(R) Kabylake")
+CHIPSET(0x5923, kbl, kbl,       "Intel(R) Kabylake")
+CHIPSET(0x5927, kbl, kbl,       "Intel(R) Kabylake")
-- 
cgit v0.10.2

